### ISSUE TITLE
Proposal to list reputable books that are possibly not free-of-charge to read

### ISSUE THREAD

**Issue by davidleejy** [rossant/awesome-math#116]
Hi, I'm raising this issue ticket to propose the possibility of listing reputable books that aren't necessarily free to read, but are commonly employed in the teaching of university-level math courses. Examples include, but aren't limited to, James Munkres' Topology, and Walter Rudin's Principles of Mathematical Analysis.

Thank you for your consideration.

**Comment by davidleejy** [comment]
If it sits well with you, I could list the books I have in mind along with categories (e.g., "Real Analysis") I think each book could be placed under in this discussion thread. This way, the public could comment on my proposed additions before I submit a pull request.

> how to differentiate them from free resources?

Unfortunately, I don't know of any particularly clever ways of achieving this aside from adding a symbol (e.g., 
**$**, 💲, 🔒), or adding a tag like [paid], [non-free], [not free] to the start or end of an entry that isn't free to read.

It's also possible, after some consideration, that we might arrive at the opinion that having a highly visible manner of differentiating paid and free resources isn't crucial in the sense that its absence doesn't impede the functionality and goal of this repository – which is a compilation of good mathematics resources which aren't necessarily free. 🤔

**Comment by rossant** [comment]
The OpenLibrary link might be sufficient since it seems to include links to buy the book, including on Amazon. That's just 1 more click to get there?

I like the idea of adding a symbol to the start of an entry for paid resources. Historically, this repository has mostly proposed free resources. We could add a note at the beginning of the page stating that all resources are assumed to be freely available, except those with a paid symbol.

For a suggested list of books to be added, perhaps you could also directly work on a pull request with your suggestions, and we can continue the discussion there?

Other folks are invited to chime in here or on that pull request. :slightly_smiling_face:

**Comment by davidleejy** [comment]
Hi Rossant, thanks for weighing in. 

> We could add a note at the beginning of the page stating that all resources are assumed to be freely available, except those with a paid symbol

> For a suggested list of books to be added, perhaps you could also directly work on a pull request with your suggestions, and we can continue the discussion there?

Ok, will submit a PR with a list and a note at the beginning 😄

**Comment by davidleejy** [comment]
Currently, the type of content (e.g., book, lecture notes) are situated at higher level than the area (e.g., analysis) and I'd like to propose swapping the levels of these two concepts by moving the type of content under the area. Roughly speaking, my proposed idea is to go from:

```md
<!-- current readme.md layout-->

## Books
...

## Lecture Notes

### Area 1
...

### Area 2
...
```

**to:**

```md
<!-- proposed readme.md layout-->

## Area 1
### Books
...
### Lecture Notes
...

## Area 2
### Books
...
### Lecture Notes
...
```

I'm proposing this in the hope that the new layout will look better overall when the list of books under each area grows.

What do you think? 😄

**Comment by rossant** [comment]
Sounds like a good idea to me! Anyone else wants to comment on this proposition?

**Comment by davidleejy** [comment]
Another idea that I'd think is worth considering is removing the type of content section header. This brings two advantages: (1) the list becomes more accomodative to all content types (lecture videos, insightful blog posts), and (2) "similar" entries can be grouped together (e.g. book and lecture notes by the same author).

Am open to comments from other people. Intend to submit a PR after a week or so.

**Comment by rossant** [comment]
Yes, I was about to suggest the same thing. Perhaps we could try adding icons at the beginning of each item, with 1 different icon per content type?

**Comment by davidleejy** [comment]
> Yes, I was about to suggest the same thing. Perhaps we could try adding icons at the beginning of each item, with 1 different icon per content type?

I thought about this too, but didn't suggest it due to the possibility of challenging situations where:
1. there aren't icons that are visually representative of some content types, or,
2. there exist different content types that are somewhat similar in nature, and there aren't icons that are represent them _and_ their differences simultaneously.

Here, I assume icons refer to emojis. But, if it is reasonable to consider also using unicode, alphabets, or a combination of these in the readme's iconography, then I think the two situations above could be readily circumvented if they arise in the future.

**Comment by rossant** [comment]
Agreed. We could start by identifying here the most common content types and determine which emojis/unicode characters etc could be used for each.

**Comment by davidleejy** [comment]
Perhaps we could start with these three icons depicting content types commonly encountered in university-level math:

📖 Books
🎥 Videos
📝 Lecture notes, slides, articles, papers

**Comment by rossant** [comment]
yes, looks good!

**Comment by LydiaCai1203** [comment]
@monkeycode-ai  what do you think

**Comment by rossant** [comment]
Thanks for the discussion. The new policy prefers free and open resources and allows exceptional paid resources. I am closing this as resolved.
